### PR TITLE
Add documentation for AVA test framework

### PR DESCRIPTION
**Pull request by maur8ino**
Hello everyone, this is my first PR! :smile: 

While trying out AVA, I've noticed that there's no reference on the site for this test framework which heavily uses Babel for transpiling test files.

Thanks!

### CODE REVIEW

**Comment by hzoo** [comment]
Awesome! Thanks for the PR

File: _includes/tools/ava/usage.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+AVA comes with built-in support for Babel 6 and let you write your tests
```
Comment (hzoo) [inline _includes/tools/ava/usage.md]: Maybe emphasize more that AVA doesn't require extra setup to use babel with es2015, but if you want you can customize it? cc @sindresorhus

Comment (maur8ino) [inline _includes/tools/ava/usage.md]: hi @hzoo I basically ripped off the text from https://github.com/sindresorhus/ava#es2015-support almost with no changes :smile:

something like this:
> AVA ships with ES2015 support for test files built-in using Babel 6 under the hood.
>
> The AVA default configuration includes the `"es2015"`and `"stage-0"` Babel 6 presets, but you can customize any Babel option for transpiling test files with the `babel` option in AVA's [`package.json` configuration](https://github.com/sindresorhus/ava#configuration).

will work?

I'm not a native speaker so please bare with me :smiley:

Thanks!

File: _data/tools.yml [hunk 2]
```diff
@@ -28,6 +28,7 @@
     sails: Sails
 - name: Test frameworks
   items:
+    ava: Ava
```
Comment (sindresorhus) [inline _data/tools.yml]: `Ava` => `AVA`

File: _includes/tools/ava/install.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+```sh
```
Comment (sindresorhus) [inline _includes/tools/ava/install.md]: `sh` => `console`

`$` makes it not a shell script. `console` works fine though.

File: _includes/tools/ava/usage.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+AVA ships with ES2015 support built-in using Babel 6 under the hood, so
+you can write test using ES1025 syntax right away.
```
Comment (sindresorhus) [inline _includes/tools/ava/usage.md]: `ES1025` - typo

File: _includes/tools/ava/usage.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    "es2015",
+    "stage-2",
+  ],
+  "plugins": [
+    "espower",
+    "transform-runtime"
+  ]
+}
+```
+but you can customize any Babel option for transpiling test files with the
```
Comment (sindresorhus) [inline _includes/tools/ava/usage.md]: `but` => `But`

File: _includes/tools/ava/usage.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    "babel": {
+      "presets": [
+        "es2015",
+        "react"
+      ]
+    }
+  },
+}
+```
+or you can simply `"inherit"` the Babel configuration from
```
Comment (sindresorhus) [inline _includes/tools/ava/usage.md]: `or` => `Or`

**Comment by maur8ino** [comment]
Done!

Thanks @sindresorhus for the review :smile:

File: _includes/tools/ava/install.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+```console
+$ npm install --save-dev babel-core
```
Comment (hzoo) [inline _includes/tools/ava/install.md]: Is this actually necessary?

Comment (sindresorhus) [inline _includes/tools/ava/install.md]: No

Comment (maur8ino) [inline _includes/tools/ava/install.md]: Ok then, I'll change it back to `sh`

Comment (sindresorhus) [inline _includes/tools/ava/install.md]: No, we're talking about the whole line. It's not necessary to install `babel-core`. AVA comes bundled with it.

Comment (maur8ino) [inline _includes/tools/ava/install.md]: @sindresorhus @hzoo sorry guys didn't catch it :disappointed: 

Should I change it with:
> There is no need to `npm install babel-core`, AVA bundles it already.

Thanks.

**Comment by maur8ino** [comment]
hi @hzoo and @sindresorhus, I've updated the `installation` part of AVA :smile:

**Comment by sindresorhus** [comment]
LGTM

**Comment by hzoo** [comment]
Ok thanks @maur8ino for sticking through with it! Next time I might ask you to squash the commits but it's all good

**Comment by maur8ino** [comment]
@hzoo no problem! :smile:

**Pull request merged**